index.html

```diff
@@ -39,7 +43,215 @@ <h1>I'm Alishah. Glad to see you!</h1>
+                        <p>A daily selection of privately <br>
+                            personalized reads;no accounts or<br>
+                            sign-ups required.</p>
+                    </div>
+                    <ul class="primarytext3">
+                        <li class="lang">html</li>
+                        <li class="lang">css</li>
+                        <li class="lang JavaScript">javascript</li>
+                    </ul>
+                    <div class="primarytext4">
+                        <button class="project">See project</button>
+                    </div>
+                </div>
+            </div>
+            <div class="works-container">
+                <div class="snapshoot">
+                    <img src="./Image/SnapshootPortfolio(5).png" alt="">
+                </div>
+                <div class="leftBlock">
+                    <div class="primarytext1">
+                        <h3> Multi-Post
+                            <br>
+                            Stories
+                        </h3>
+                        <div class="fram1">
+                            <span class="canopy">CANOPY</span>
+                            <img src="./Image/Counter.png" alt="center">
+                            <span class="num-text">Back End Dev</span>
+                            <img src="./Image/Counter.png" alt="center">
+                            <span class="num-text">2015</span>
+                        </div>
+                    </div>
+                    <div class="primarytext2">
+                        <p>A daily selection of privately <br>
+                            personalized reads;no accounts or<br>
+                            sign-ups required.</p>
+                    </div>
+                    <ul class="primarytext3">
+                        <li class="lang">html</li>
+                        <li class="lang">css</li>
+                        <li class="lang JavaScript">javascript</li>
+                    </ul>
+                    <div class="primarytext4">
+                        <button class="project">See project</button>
+                    </div>
+                </div>
+            </div>
+
+        </section>
+        <section class="about-my-slef">
+            <div class="top">
+                <div class="text-container">
+                    <h2 class="first-text">About <br> Myself</h2>
+                    <p class="second-text">Hello I'm a software developer! I can help you <br>
+                        bulid a product, feature or website.<br>
+                        Look through some of my work and <br>
+                        experience! if you like what you see and <br>
+                        have a project you need coded, don't <br>
+                        hesitate to contact me.
+                    </p>
+                </div>
+
+                <div class="social-media1">
+                    <h3 class="sub-social-media ">LET’S CONNECT</h3>
```

Comment:
- [ ] [OPTIONAL] Once again I'd like to apologize as the previous reviewer forgot to mention this but the "LET’S CONNECT" is a link so it should also be inside an `a` tag.
    - You can place the `h3` inside an `a` tag or vise versa